.github/workflows/build.yml

```diff
@@ -1,6 +1,7 @@
 name: Build
 
 on:
+  workflow_dispatch: {}
   push:
     branches:
       - "main"
```

Comment:
We should probably remove this "main" filter too.

Reply:
why?